hardware_layer/hardware_arduino/launch/hardware_nodes.launch

```diff
@@ -0,0 +1,8 @@
+<launch>
+    <!--<node name="yawDirect" pkg="hardware_imu" respawn="true" type="yawDirect"/>-->
+    <!--<node name="serial_node" pkg="hardware_arduino" respawn="true" type="serial_node.sh"/>-->
+    <node name="cameras" output="screen" pkg="hardware_camera" respawn="true" type="vid_pub">
```

Comment:
Why is camera node being run from here? Doesn't it have its own launch file?

Reply:
Removed the camera node code , and modified it .....